configure.ac

```diff
@@ -135,29 +122,14 @@ VA_MINOR_VERSION=`echo "$VA_VERSION" | cut -d'.' -f2`
 VA_MICRO_VERSION=`echo "$VA_VERSION" | cut -d'.' -f3`
 VA_VERSION_STR="$VA_VERSION"
 
-va_full_version_int=`expr ${VA_MAJOR_VERSION:-0} "*" 1000000 + \
-                          ${VA_MINOR_VERSION:-0} "*" 10000   + \
-                          ${VA_MICRO_VERSION:-0} "*" 100     + \
-                          0`
 VA_DRIVER_INIT_FUNC="__vaDriverInit_${VA_MAJOR_VERSION}_${VA_MINOR_VERSION}"
 AC_DEFINE_UNQUOTED([VA_DRIVER_INIT_FUNC], [$VA_DRIVER_INIT_FUNC],
     [Define driver entry-point])
 
-dnl Check for VA/DRM API
+dnl Check for VA/X11 API
 USE_X11="$enable_x11"
-
-dnl Check for X11
-if test "$enable_x11" = "yes"; then
-    USE_X11="yes"
-    PKG_CHECK_MODULES([X11],    [x11],    [:], [USE_X11="no"])
-    PKG_CHECK_MODULES([XEXT],   [xext],   [:], [USE_X11="no"])
-    PKG_CHECK_MODULES([XFIXES], [xfixes], [:], [USE_X11="no"])
-fi
-
 if test "$USE_X11" = "yes"; then
-    PKG_CHECK_MODULES(LIBVA_X11_DEPS, [libva-x11],
-      [AC_DEFINE([HAVE_VA_X11], [1], [Defined to 1 if VA/X11 API is enabled])],
-      [USE_X11="no"])
+   AC_DEFINE([HAVE_VA_X11], [1], [Defined to 1 if VA/X11 API is enabled])
```

Comment:
We still need to check for libva-x11 availability.  If libva is compiled with `--disable-x11` the associated libva-x11 headers don't get installed.  Then, when user tries to `--enable-x11` on intel-vaapi-driver, the compilation will fail due to missing headers.   It's better to fail during configure if libva-x11 is not installed.